### PR TITLE
chore: Add Catalina to WDA

### PR DESCRIPTION
**Pull request by dpgraham**
* Add Catalina bundlers
* Add more documentation to explain how bundling works

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]

File: ci-jobs/build.yml [hunk 1]
```diff
@@ -9,4 +9,9 @@ jobs:
           addChangeLog: false
   - template: ./templates/build.yml
     parameters:
-      name: 'macOS_10_14'
\ No newline at end of file
+      excludeXcode: '11.1,11.2.1,11.2,11.3.1,11.3,11'
```
Comment (mykola-mokhnach) [inline ci-jobs/build.yml]: why it is necessary to explicitly exclude these versions? Azure selects the most recent version by default, doesn't it?

**Review by jlipps — APPROVED** [review]

File: docs/CREATING_BUNDLES.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Creating WebDriverAgent bundles using Azure Pipelines
+
+The [bundle](https://dev.azure.com/AppiumCI/Appium%20CI/_build?definitionId=36&_a=summary) pipeline uses macOS agents on Azure and iterates through every installed version of Xcode and then builds WebDriverAgent using ./scripts/build-webdriveragents.js
+
+The bundle pipeline gets runs every time there is a tag and builds to a folder called `prebuilt-agents` and those agents get uploaded to [GitHub Releases](https://github.com/appium/WebDriverAgent/releases).
```
Comment (imurchie) [inline docs/CREATING_BUNDLES.md]: gets run

**Comment by Arrognz** [comment]
Hello :) 

Can you explain how Signing works with prebuilt WDA ? I see in the source code that you update bundle ID, that you link a xcconfig file during `test-without-building`. However, is that all it takes to use the prebuilt WDA bundles ? Also, can you use prebuilt WDA with real devices ? 

I cannot find documentation about using prebuilt WDA, is it missing or am I just bad at finding it ?

Thank you for your time !

**Comment by KazuCocoa** [comment]
Prebuild module works only on a simulator since a real device requires its own signing, or rebuild module specialised for a device.

File: ci-jobs/build.yml (hunk 1 above)
Comment (dpgraham) [reply ci-jobs/build.yml]: @mykola-mokhnach sorry for the super late reply.

We only want one build per iOS version so we exclude versions from 10.14 that are included in 10.15

Comment (dpgraham) [reply ci-jobs/build.yml]: I'll add a comment.

**Comment by dpgraham** [comment]
@ganzf you can open the Xcode Project and add your own signing certifactes to the project and build an IPA from it. Unfortunately, on our side, we can't do prebuilt IPA's (only .app).

If you come up with a way to do that though, I'd love to hear it! Prebuilt IPA's would be amazing.

**Review by jlipps — APPROVED** [review]

**Pull request merged**